types/database/models/servers/post.d.ts

```diff
@@ -57,19 +57,19 @@ export default class PostModel extends Model {
     props: object;
 
     /** drafts  : Every drafts associated with this Post */
-    drafts: DraftModel;
+    drafts: Query<DraftModel>;
```

Comment:
Why is draft a Query and not a Relation?

Reply:
cause the draft will only match on root_id and not the id, also in the model is defined as a lazy query